app/views/streams/_job_status.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="mb-5">
   <h3><%= t('job_tracker.title') %></h3>
   <div class="accordion" id="jobStatusAccordian">
-    <% stream.job_tracker_status_groups.each do |status_group, jobs| %>
+    <% needs_attention = stream.job_tracker_status_groups.select { |sg| sg == :needs_attention } %>
+    <% status_groups_ordered = needs_attention.merge(stream.job_tracker_status_groups) %>
```

Comment:
Maybe set the order of the status groups in the model so you don't have to mess with it in the view?:

https://github.com/pod4lib/aggregator/blob/main/app/models/stream.rb#L30

Reply:
OH wow that was so much easier! Thank you